components/AvalancheProblemLikelihoodLine.tsx

```diff
@@ -16,19 +16,14 @@ export const likelihoodText = (input: AvalancheProblemLikelihood): string => {
 };
 
 const likelihoodToRange = (likelihood: AvalancheProblemLikelihood): SeverityNumberLineRange => {
-  let index: number = 4;
-  switch (likelihood) {
-    case AvalancheProblemLikelihood.AlmostCertain:
-      index -= 1;
```

Comment:
eslint didn't like fall-through here (most lint tools don't in my experience 😄)

Reply:
Wild. Poorly supported feature of the language?

Reply:
I think accidental fallthrough in switch statements has been a concern since I first learned C, and that was back in the K&R days 👴. Like yeah, it's valid but it's often not what people intend! In this case it was easy enough to rewrite it.

Looks like C++17 lets you annotate this explicitly now, cool! https://en.cppreference.com/w/cpp/language/attributes/fallthrough

Reply:
I'm used to explicit fallthrough in Go!

Reply:
ah cool, I haven't learned to speak Go yet but it's on my list 😄